rust/xaynet-server/src/storage/redis.rs

```diff
@@ -169,14 +171,14 @@ impl Connection {
     /// Adds a new [`SumDict`] entry.
     ///
     /// Returns [`Ok(())`] if field is a new or
-    /// [`SumDictAddError::AlreadyExists`] if field already exists.
+    /// [`SumPartAddError::AlreadyExists`] if field already exists.
     ///
-    /// [`SumDictAddError::AlreadyExists`]: [crate::storage]
+    /// [`SumPartAddError::AlreadyExists`]: [crate::storage]
     pub async fn add_sum_participant(
         mut self,
         pk: &SumParticipantPublicKey,
         ephm_pk: &SumParticipantEphemeralPublicKey,
-    ) -> RedisResult<SumDictAdd> {
```

Comment:
The new name feels more correct to me. But at first it just confused me because I didn't directly understand that "part" was short for "participant". But I understand that `SumParticipantAddError` is annoyingly long...